Automattic/wp-calypso
# Sensei Tailored Flow: Add 'Choose the purpose of your site' step

Depends on https://github.com/Automattic/wp-calypso/pull/72473

#### Proposed Changes

* Add a new step after checkout to the Sensei flow, replicating the plugin onboarding screen
* Show dummy progress bar to pace out waiting for the atomic site
* Remove launchpad from steps

https://user-images.githubusercontent.com/176949/214653168-195835af-1c2d-46ba-9a13-2e45fc6b0bdf.mov

#### Testing Instructions

* Create a new site via http://calypso.localhost:3000/setup/sensei/
* After checkout, it should show the new screen
* To test just the screen without having to create a new site, use this link with an existing site URL and ID: `calypso.localhost:3000/setup/sensei/senseiPurpose?siteSlug= [URL] &siteId= [ID]`

#### Pre-merge Checklist





## Screenshots for translation

    Order was completed successfully
    Preparing your new site

<img width="800" alt="image" src="https://user-images.githubusercontent.com/176949/217371611-f021b52b-6be0-4bf4-ba85-1d92ad7e695c.png">

    Choose the purpose of your site
    Select your goals for offering courses, and we will help you set everything up.
    Sell courses and generate income
    Provide certification
    Educate students
    Train employees

<img width="802" alt="image" src="https://user-images.githubusercontent.com/176949/217371887-df333486-17f2-4752-9993-ab03dd32cb8d.png">
